modules/@apostrophecms/image/ui/apos/components/AposMediaManagerDisplay.vue

```diff
@@ -31,6 +31,9 @@
             v-model="checkedProxy"
           />
         </div>
+
+        <slot name="actions" />
+
         <button
```

Comment:
Why not putting the placeholder slot around this button? It would allow to have the border on the actual parent of the card, and would be easier to style.